website/sidebars.js

```diff
@@ -114,7 +114,7 @@ module.exports = {
         "current/api/build-custom-client",
         {
           type: "link",
-          label: "Reference",
+          label: "Reference 🔗",
```

Comment:
I don't think this is needed. We use the chain icon to indicate an external resource (for Go and Python) but the API docs and the Node.js docs are hosted by us so do not require this icon IMO.